<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频播放器:QTversion--powered by zlp2023</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LB_current">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>播放进度（时间）</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_total">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>播放总时长</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QSlider" name="SL_sound">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>510</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="SL_progress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>461</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>浏览器</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>播放列表</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>531</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>默认专辑</string>
       </attribute>
       <widget class="QListWidget" name="L_MediaList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>451</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>470</y>
          <width>21</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>470</y>
          <width>21</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>470</y>
          <width>21</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>470</y>
          <width>21</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="B_AddFile">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>470</y>
          <width>41</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>470</y>
          <width>41</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>470</y>
          <width>41</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>排序</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>470</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>470</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>此电脑</string>
       </attribute>
       <widget class="QListWidget" name="L_UserFile">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>451</width>
          <height>491</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="B_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_stop">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>510</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_prev">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>上一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_next">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>510</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>下一</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_playwidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
